CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [1.2.0] - 2021-07-16
+### Added
+- Print out PKCS7 certs with auth variable verification fails.
```

Comment:
Is there a typo? Assuming that 'fails' is a verb I have trouble understanding this line.

Reply:
with -> when.  Fixed.